src/views/become-a-scratcher/become-a-scratcher.scss

```
@@ -0,0 +1,476 @@
+        }
+    }
+
+    .congratulations-text-layout{
+        flex: 1;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        flex-direction: column;
+        padding: 40px;
+        justify-content: space-between;
+    
+        @media #{$medium-and-smaller} {
+            padding-top: 0px;
+        }
+    
+        .center-text{
+            display: flex;
+            flex: 1;
+            flex-direction: column;
+            justify-content: center;
+        }
+
+        .congratulations-text{
+            display: flex;
+            flex-direction: column;
+            justify-content: center;
+            align-items: center;
+            max-width: 700px;
+            margin: auto;
+        }
+    }
+    .congratulations-image-layout{
+        flex: .9;
+        display: flex;
+        position:relative;
+        justify-content: center;
+        background-color: #E9F1FC;
+        @media #{$medium-and-smaller} {
+            width: 100vw;
+            order: -1;
+            margin-top: -90px;
+        }
+        .congratulations-image-container{
+            position: relative;
+            display: flex;
+            margin: auto;
+            height: 375px;
+        }
+    }
+    .congrats-banner-image{
+        width: 328px;
+    }
+}
+
+
+
+.invitation-card{
+    background-color: white;
+    margin: 10px;
+    border-radius: 7px;
+    padding: 40px;
+    text-align: center;
+    max-width: 600px;
+
+    .invitation-image{
+        margin-bottom: 20px;
+    }
+
+    .invitation-icon{
+        border-radius: 3px;
+        width: 50px;
+        margin: 0px 10px;
+    }
+
+    button{
+        margin: 10px;
+        margin-top: 30px;
+    }
+
+    .invitation-buttons{
+        display: flex;
+        justify-content: center;
+    }
+}
+
+.blue-background{
+    background-color: #3373cc;
+}
+
+.labeled-icon{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    
+    img{
+        margin-right: 20px;
+    }
+}
+
+.navigation{
+    display: flex;
+    width: 100%;
+    justify-content: space-around;
+    align-items: center;
+    margin-bottom: 40px;
+
+    .dotRow{
+        display: flex;
+        justify-content: space-between;
+        width: 244px;
+        min-width: 0px;
+        padding: 0px 5px;
+    }
+
+    .dot{
+        width: 20px;
+        height: 20px;
+        border-radius: 100%;
+        background-color: #d9d9d9;
+        border: none;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+    }
+    .active{
+        border: solid 2px #4280d9;
+        background-color: #4c97fe;
+        margin-top: -2px;
+    }
+
+    .navText{
+        @media #{$small} {
+            display: none;
+        }
+    }
+}
+
+
+.onboarding {
+    display: flex;
+    flex-direction: row;
+    min-height: 100vh;
+    line-height: 1.8rem;
+    font-size: 1.1rem;
+
+    h1{
+        line-height: 3rem;
+        font-size: 32px;
```

We typically use rem for font size
```suggestion
        font-size: 2rem;
```